### ISSUE TITLE
Chart data not updating

### ISSUE THREAD

**Issue by Evanslooten** [ConsoleTVs/Charts#662]
I was playing around with the package to see if it might be a good solution for my project, followed the documentation, and got a test chart rendered. Great! Then when I modified the data set, the chart did not reflect the changes. I cleared cache, routes, and views, but still, nothing I do in the App\Charts\TestChart file would reflect in the view. 

If I create some manual routes, one for API and one for rendering the view, the Chartisan package does work properly. What am I doing wrong with this package?

**Comment by ConsoleTVs** [comment]
Can you paste how you update the chart on the client?

**Comment by Evanslooten** [comment]
All I did was create a test chart with the make command, then changed the sample dataset that was being returned as a chartisan instance. Changing the values or even commenting out the return would not modify the chart on refresh. 

I opted to make my own files and just use the Chartisan package. I now have an issue where I removed the Charts I created and now I cannot remove the package. When I attempt composer remove consoletvs\charts I get `Target class [ConsoleTVs\Charts\Registrar] does not exist.`
